src/cockpit/superuser.py

```diff
@@ -147,6 +147,7 @@ def __init__(self, router: Router, *, privileged: bool = False):
         self.pending_prompt = None
         self.peer = None
         self.startup = None
+        self.initing_name: Optional[str] = None
```

Comment:
I feel like we've been around this one before and decided last time to rather look at `peer.name` or whatever, no?